python/lsst/ts/cbp/component.py

```diff
@@ -1,140 +1,119 @@
 
-    panic_status: float
+    panic_status : `float`
         The current value of the panic variable in the CBP dmc code.
-        A non-zero value represents a panic state and causes the motors to cease functioning until panic is
-        dealt with or goes away.
-        This status is related to the other statuses.
+        A non-zero value represents a panic state and causes the motors to
+        cease functioning until panic is dealt with or goes away.
+        This status is set by the values of the other statuses.
 
-    auto_park: float
+    auto_park : `float`
         The current value of the auto_park variable.
-        If this value is one, that means that CBP suffered a power loss that lasted more than 12 seconds and
-        was on battery back up.
-        The CBP will then park itself automatically, moving azimuth to 0 and altitude to -70 and lock focus
-        and mask.
-        To unpark CBP, the park variable should be set to zero.
+        If this value is one, that means that CBP suffered a power loss that
+        lasted more than 12 seconds and was on battery back up.
+        The CBP will then park itself automatically, moving azimuth to 0 and
+        altitude to -70 and lock focus and mask.
+        To un-park CBP, the park variable should be set to zero.
 
-    park: float
+    park : `float`
         The current value of the park variable.
-        This value can be set to one or zero, if set to one it will park the CBP if set to zero it will
-        unpark.
-
-    azimuth_status: float
-        This is the current value of the status of the azimuth encoder.
-        If this value is non-zero, the encoder has an :ref:`error<intro:Errors>`.
-
-    altitude_status: float
-        This is the current value of the status of the altitude encoder.
-        If this value is non-zero, the encoder has an :ref:`error<intro:Errors>`.
+        This value can be set to one or zero, if set to one it will park the
+        CBP if set to zero it will un-park.
 
-    mask_select_status: float
-        This is the current value of the status of the mask selection encoder.
-        If this value is non-zero, the encoder has an :ref:`error<intro:Errors>`.
-
-    mask_rotate_status: float
-        This is the current value of the status of the mask rotation encoder.
-        If this value is non-zero, the encoder has an :ref:`error<intro:Errors>`.
-
-    focus_status: float
-        This is the current value of the status of the focus encoder.
-        If this value is non-zero, the encoder has an :ref:`error<intro:Errors>`.
+    encoder_status : `Status`
+        A flag enum that contains the status information for each encoder.
 
     Notes
     -----
 
-    The class uses the python socket module to build TCP/IP connections to the galil controller mounted onto
-    CBP.
+    The class uses the python socket module to build TCP/IP connections to the
+    galil controller mounted onto CBP.
```

Comment:
Please consider changing "mounted onto the CBP" to "for the CBP" or perhaps "at the CBP" (the location of the Galil is an internal detail; what is crucial is that it is the Galil that controls the CBP).

Also please do a global search for "galil" and change it to "Galil" -- it's a company name -- being careful about anchors (which you may not want to change).